controllers/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -326,6 +329,26 @@ func (r *NodeNetworkConfigurationPolicyReconciler) SetupWithManager(mgr ctrl.Man
 	return nil
 }
 
+func (r *NodeNetworkConfigurationPolicyReconciler) forceNNSRefresh(name string) {
+	log := r.Log.WithName("forceNNSRefresh").WithValues("node", name)
+	log.Info("forcing NodeNetworkState refresh after NNCP applied")
+	nns := &nmstatev1beta1.NodeNetworkState{}
+	err := r.Client.Get(context.TODO(), types.NamespacedName{Name: name}, nns)
+	if err != nil {
+		log.Error(err, "failed retrieving NodeNetworkState to force refresh")
+		return
+	}
+	if nns.Labels == nil {
+		nns.Labels = map[string]string{}
+	}
+	nns.Labels[state.ForceRefreshLabel] = fmt.Sprintf("%d", time.Now().UnixNano())
```

Comment:
this label should be removed when the nns is updated?

Reply:
The first idea was about a label without value and remove at the end of Reconcile on NNS but if the Reconcile fail the label is going to be there and there is not going to be a way to detect force refresh if we compare only the label existence, that's why I just keep it with a timestamp so we know whene the NNS was force refreshed last time (not a bad thing to have though).

Reply:
Gotcha, thanks